src/gql/fragments/base-task.graphql

```diff
@@ -4,6 +4,16 @@ fragment BaseTask on Task {
   id
   patchNumber
   status
+  tests(opts: null) {
```

Comment:
This would try to fetch test results for task logs where they aren't really relevant. This should only be requested when we fetch test logs.